views/event.ejs

```diff
@@ -647,7 +822,92 @@
             </div>
         </div>
     </div>
+
+    <div class="modal propose-session-modal" id="propose-session-modal" role="dialog" aria-hidden="true">
+
+        <div class="modal-dialog">
+            
+            <div class="modal-content">
+                
+                <div class="modal-header">
+                    <h4 class="modal-title">Propose a Session</h4>
+                </div>
+
+                <div class="modal-body">
+                    <div class="row modal-row">
+                        
+                        <div class="input-topic-box col-lg-12 no-padding">
+
+                            <div class="col-lg-12 no-padding">
+                                <div class="form-group">
+                                    <label class="col-lg-3 control-label no-padding session-title-label" for="topic_title">Session Title</label>
+
+                                    <div class="col-lg-9 no-padding">
+                                        <input class="form-control input-topic-title" type="text" name="topic_title" placeholder="Enter session title in less than 80 characters"value="" id="topic_title">
+                                    </div>
+                                </div>
+                            </div>
+
+                            <div class="col-lg-12 no-padding">
+                                 <hr/> 
+                            </div>
+                        </div>
+
+                        <div class="session-preview col-lg-12 col-md-12 col-sm-12 col-xs-12">
+
+                        <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12 proposed-session-box">
```

Comment:
Please don't duplicate column classes like this -- I've made this comment before, but ``col-lg-12 col-md-12 col-sm-12 col-xs-12`` is exactly the same as ``col-xs-12`` and is harder to read.  Bootstrap is modal-first: it starts with the smallest style and works its way up.  If small, medium, or large are different than xs great -- e.g. ``col-lg-9 col-md-10 col-sm-11 col-xs-12``.  But otherwise, only use the classes that will actually change the appearance.